src/features/twinStatus/selectField/twin-status-select-field.tsx

```diff
@@ -13,6 +18,18 @@ type Props<T extends FieldValues> = {
 export function TwinStatusSelectField<T extends FieldValues>(props: Props<T>) {
   const { getById, getItems, getItemKey, getItemLabel } =
     useTwinStatusSelectAdapter(props.twinClassId);
+  const { fetchTwinStatusById } = useFetchTwinStatusById();
+  const statusId = useWatch({
+    control: props.control,
+    name: props.name,
+  });
+  const [initVals, setInitVals] = useState<TwinStatus[]>([]);
```

Comment:
Need to refactor after https://alcosi.atlassian.net/browse/TWINFACES-207 - it is combobox's (or select field's) responsibility to fetch object value from field value (id).